metricflow/test/api/test_metricflow_client.py

```diff
@@ -93,6 +93,32 @@ def test_list_dimensions(mf_client: MetricFlowClient) -> None:  # noqa: D
     assert tuple(dim.name for dim in dimensions) == ("metric_time", "metric_time")
 
 
+def test_get_measures_for_metrics(mf_client: MetricFlowClient) -> None:  # noqa: D
+    measures = mf_client.engine.get_measures_for_metrics(["bookings"])
+    assert measures
```

Comment:
Doesn't much matter, but maybe `assert len(measures) == 1` just for consistency?